src/models/sensor-definitions.ts

```diff
@@ -347,7 +375,7 @@ export const SensorDefinitions:ISensorDefinitions = {
     "minReading": 0.0,
     "maxReading": 0.5
   },
-  "%RH": {
+  "%_RH": {
```

Comment:
I think this needs to remain `%RH` because I'd guess there is some other sensor that returns this unit.